backend/store/postgres.go

```diff
@@ -26,132 +32,155 @@ type CollaborationStoreImpl GormStore
 
 type SrsStoreImpl GormStore
 
-func NewPostgresStore() (*gorm.DB, error) {
+func NewPostgresStore(ctx context.Context) (*gorm.DB, error) {
 	_ = godotenv.Load("../.env", "../.env.example")
 	host := os.Getenv("POSTGRES_HOST")
 	username := os.Getenv("POSTGRES_USER")
 	password := os.Getenv("POSTGRES_PASSWORD")
 	dbname := os.Getenv("POSTGRES_DB")
 	port := os.Getenv("POSTGRES_PORT")
 
+	logger := logger.New(
+		logrus.NewWriter(),
+		logger.Config{
+			SlowThreshold: time.Millisecond,
+			LogLevel:      logger.Warn,
+			Colorful:      false,
+		},
+	)
+
 	connStr := fmt.Sprintf("postgres://%s:%s@%s:%s/%s?sslmode=prefer", username, password, host, port, dbname)
-	return gorm.Open(postgres.Open(connStr), &gorm.Config{})
+	db, err := gorm.Open(postgres.Open(connStr), &gorm.Config{Logger: logger})
+	if err != nil {
+		return nil, err
+	}
+	
+	err = db.Use(tracing.NewPlugin(
+		tracing.WithDBName("kioku"),
+	))
```

Comment:
```suggestion
	if err = db.Use(tracing.NewPlugin(tracing.WithDBName("kioku"))); err != nil {
		return nil, err
	}
```
Maybe you can return the error already here, so that in the general return of the function a `return db.WithContext(ctx), nil` could be used.

Reply:
yup yup!
fixed in [2d123c2](https://github.com/kioku-project/kioku/pull/196/commits/2d123c252b7c4497f5debae39d174546684f3782)